### DISCUSSION TITLE
Older database dumps?

### DISCUSSION THREAD

**Discussion by avdi** [rubygems/rubygems.org#2952] [Q&A] [answered]
Heya, I'm setting up for a historical code demo thing, using a 2016 version of rubygems.org. I'd like to populate it, but the db dumps listed at https://rubygems.org/pages/data only go back into 2021. 

Are by any chance older dumps available anywhere?

Thanks!

**Comment by sonalkr132** [comment]
Hi, we don't have dumps older than the ones listed on the page.

**Comment by sonalkr132** [comment, marked as answer]
correction, we have the backup from 2014-2019, however, it is not the public dump and is encrypted. it needs a significant amount of work to decrypt and expunge.

**Reply by avdi** [reply]
Thanks for looking!
